src/supertux/menu/game_menu.cpp

```diff
@@ -19,11 +19,14 @@
 #include "gui/menu.hpp"
 #include "gui/menu_item.hpp"
 #include "gui/menu_manager.hpp"
+#include "gui/dialog.hpp"
+#include "gui/mousecursor.hpp"
 #include "supertux/game_session.hpp"
 #include "supertux/level.hpp"
 #include "supertux/menu/menu_storage.hpp"
 #include "supertux/menu/options_menu.hpp"
 #include "supertux/screen_manager.hpp"
+#include "supertux/gameconfig.hpp"
```

Comment:
Those includes are still not sorted alphabetically. gui/dialog is before gui/menu. gameconfig before game_session.